Initial run of agents at "Getting Started" section fails

## Description
<!--- Tell us what happens instead of the expected behavior -->
Following "Getting Started" section of README on a fresh Linux install. Error occurs at step 5 of "Getting Started". Output below:
```python
set -a && . ./.env && set -a && PYTHONPATH=. python3 orchestrator/runner.py
[Pipeline] Running Virology Agent...
[RAG] No RAG index found at: data/rag_indexes/virology_rag
[ERROR] Pipeline execution failed: 'NoneType' object has no attribute 'retrieve'
```

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
There seems to be checking directory `data/rag_indexes` for file `virology_rag`. If not found or file is empty, process continues and tries to read or retrieve information from "None" object.
Disabling the following code block on `agents/base_agent.py` allows the the `make run` command to finish and export the expected output:
```python
if hasattr(self, "rag_retriever"):
            documents = self.rag_retriever.retrieve(self.variables)
            rag_context = "\n".join([doc.page_content for doc in documents])
            self.variables["context"] = rag_context
```
Possible solution is to check if the `documents` variable has a valid type before trying to retrieve its information.

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Follow steps of "Getting Started" section.

## Context (Environment)
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->
- OS: Manjaro Linux 25.0.0
- Python version: 3.13.2

--- a/agents/base_agent.py
+++ b/agents/base_agent.py
@@ -96,7 +96,7 @@ class BaseAgent:
         prompt = self.load_prompt()
 
         # Always retrieve documents from RAG retriever and inject into context
-        if hasattr(self, "rag_retriever"):
+        if self.rag_retriever and hasattr(self, "rag_retriever"):
             documents = self.rag_retriever.retrieve(self.variables)
             rag_context = "\n".join([doc.page_content for doc in documents])
             self.variables["context"] = rag_context

